composer.json

```diff
@@ -21,6 +21,7 @@
 		"doctrine/dbal": "^2.10",
 		"guzzlehttp/guzzle": "^6.5",
 		"league/flysystem": "^1.0",
+		"league/mime-type-detection": "^1.0",
```

Comment:
`league/flysystem` is shipped with `mime-type-detection` (it's upgraded to a separate package) but I wanted to be explicit.